pkg/v1/remote/transport/error.go

```diff
@@ -89,8 +89,13 @@ func (e *Error) Temporary() bool {
 		return false
 	}
 	for _, d := range e.Errors {
-		// TODO: Include other error types.
-		if d.Code != BlobUploadInvalidErrorCode {
+		isTempCode := false
+		for _, code := range temporaryErrorCodes {
```

Comment:
I think this would be easier to read if we turned temporaryErrorCodes into a set, wdyt?

Reply:
done

Reply:
Thanks!